tutorials/fitter2/chi2_example.ini

```diff
@@ -0,0 +1,123 @@
+# Data config
+# Required
+[data sets]
+zeff = 2.334
+ini files = [path to config]
+
+##########
+
+# Choose if you want to model isotropic or anisotropic BAO
+# ap_at or aiso_epsilon
+# Required
+[cosmo-fit type]
+cosmo fit func = ap_at
+
+##########
+
+# Fiducial Pk file
+# Required
+[fiducial]
+filename = PlanckDR16/PlanckDR16.fits
+
+##########
+
+# Choose verbosity
+# Optional
+[verbosity]
+level = 0
+
+##########
+
+# Control Section
+# Needed when calling fitter2_control and fitter2_control_mpi
+# Optional
+[control]
+# General run for the fitter. This is True for everything except when you run Polychord
+chi2 = True
+
+# Parallel version of the fitter. Works for the Scan and fastMC
+# Only works if you call fitter2_control_mpi
+# chi2 still needs to be true above
+chi2_parallel = False
+
+# Run Polychord. The other two options should be false.
+# Only works if you call fitter2_control_mpi
+# This can be very slow if not run on multiple cores
+sampler = False
+
+##########
+
+# Path for the chi2 output. This should generally be an .h5 file
+# Required
+[output]
+filename = [path to output file].h5
+
+##########
+
+# Scan config. Start, Stop, Steps
+# One or two parameters only
+# Optional
+[chi2 scan]
+ap = .95 1.15 15
+at = .85 1.15 15
```

Comment:
I think this example is great, but it might take a while to run when not using MPI. 

I wonder whether we should have also an example file that can test the different functionalities, but that would run in a couple of minutes. Doing a scan on a single parameter, doing 10 FastMC mocks, and so on. 

We can open an issue about this, no need to fix everything at the same time.

Reply:
I reduced the number of runs, so hopefully, it should be doable in a few minutes now. However, it all depends on how many free parameters we have.